spec/lib/catalog/rbac/access_spec.rb

```diff
@@ -68,13 +67,7 @@
       context "when the user is in the group scope" do
         let(:scopes) { %w[group user] }
 
-        before do
-          allow(aceable_type).to receive(:try).with(:supports_access_control?).and_return(supports_access_control?)
-        end
-
         context "when the class supports access control" do
```

Comment:
Now that we removed the converse of this, we can probably remove this context so it's not as confusing later when we come in and go "wait why does this context exist? where's the 'when it doesn't support access control' context?"

Reply:
@eclarizio I have removed it